.changeset/rich-fans-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/slang": patch
+---
+
+Use correct versions for the `revert` and `global` contextual keywords
```

Comment:
nit: I suggest removing "contextual", as that term is not used anywhere else in the grammar/project.

```suggestion
Use correct versions for the `revert` and `global` keywords.
```